### PR TITLE
Missing Java classes

### PR DESCRIPTION
**Pull request by spartantri**
Added a few missing java classes
944120 |getproperty|filewriter|xmldecoder
944220 |getproperty|filewriter|xmldecoder
944240 |getproperty|filewriter|xmldecoder

java-classes.data:
+java.io.FilePermission
+java.io.FileWriter
+java.beans.XMLDecoder

### CODE REVIEW

**Comment by lifeforms** [comment]
Needs rebasing, but the Java additions are OKd by @franbuehler @lifeforms.
Quick merge after the conflicts are fixed.

File: crs-setup.conf.example [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # ------------------------------------------------------------------------
-# OWASP ModSecurity Core Rule Set ver.3.0.2
+# OWASP ModSecurity Core Rule Set ver.3.2.0
 # Copyright (c) 2006-2017 Trustwave and contributors. All rights reserved.
 #
 # The OWASP ModSecurity Core Rule Set is distributed under
```
Comment (fgsch) [inline crs-setup.conf.example:5]: The 3 changes above here should be done separately.

**Comment by lifeforms** [comment]
Hi @spartantri I'd like to merge this contribution, the core of your changes is fine, but this PR has some conflicts probably due to unimportant files... What do you think? Would you prefer to clean this up, start a new PR, or have somebody else make the PR?

**Comment by spartantri** [comment]
Let me fix the conflicts to push this one

**Comment by spartantri** [comment]
@lifeforms  there are failing checks in travis not related to this change, there are no conflicts, some of the changes to 944 where already merged as part of other PR I guess so I simply replace mine with the current version

**Comment by fgsch** [comment]
Can you just rebase the core of the changes against v3.2/dev?

**Comment by lifeforms** [comment]
Since the Git history for this PR is now messy due to changes in base, @spartantri will make a new PR and we will close this one.

**Pull request closed without merging**